packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -769,6 +769,22 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         return (currentActiveRequests() && getVotePhase() == Phase.Reveal);
     }
 
+    function _computePendingStakes(address wallet, uint256 amount) internal override {
+        if (getVotePhase() != Phase.Reveal) return;
+        uint256 currentRoundId = getCurrentRoundId();
+        bool inActiveReveal = false;
+        // If we are during an active reveal phase then we store the pending stake for each price request.
+        for (uint256 i = 0; i < pendingPriceRequests.length; i = unsafe_inc(i)) {
+            if (_getRequestStatus(priceRequests[pendingPriceRequests[i]], currentRoundId) == RequestStatus.Active) {
+                if (!inActiveReveal) {
+                    inActiveReveal = true;
+                    _freezeRoundVariables(currentRoundId);
```

Comment:
Only calls `_freezeRoundVariables` once if `getVotePhase() == Phase.Reveal` and there is at least one `active` price request.